src/main/java/hudson/remoting/Checksum.java

```diff
@@ -19,7 +20,7 @@
  *
  * @author Kohsuke Kawaguchi
  */
-final class Checksum {
+final class Checksum implements Serializable {
```

Comment:
?

Reply:
Comes from one of the combination of `hudson.remoting.PrefetchingTest`, which I guess was not tested before.